frontend/src/metabase/sw.js

```diff
@@ -0,0 +1,36 @@
+import { registerRoute } from "workbox-routing";
+import {
+  StaleWhileRevalidate,
+  NetworkFirst,
+  NetworkOnly,
+} from "workbox-strategies";
+
+// network first for HTML pages
+registerRoute(
+  ({ url }) => url.pathname === "/" || url.pathname === "/app/",
+  new NetworkFirst(),
```

Comment:
I'm realising with some extra work, we'd be able to have that embedded dashboard load offline, which is a big deal! Would be very useful in the districts.